README.md

```diff
@@ -1 +1,79 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Hello Microverse
+
+> Hello Microverse project is built for training and understanding Linters and stylelint.
+
+## Built With
+
+- HTML, CSS
+- Git, Github
+- VSCode
+
+## Getting Started
+
+Clone the folder to your local machine by using ,(https://github.com/muhammedb8826/Hello-world.git) link
+
+### Prerequisites
+
+### Setup
+Set up the yaml file located in <code>.github/workflows/linters.yml</code>
+
+### Install
+Install all different dependencies of linters file:
+- lighthouse (<code>.github/workflows/linters.yml</code>)
+- webhint (<code>.github/workflows/linters.yml</code>)
+- stylelint (<code>.github/workflows/linters.yml</code>)
+
+### Run tests
+After pushing the changes on github, merge your pull request and let github action process changes.
+
+### Deployment
+Check for the tests when you generate a pull request and fix errors, if any.
+For stylelint errors run:<code>sudo npx stylelint "**/*.{css,scss}" --fix</code> for automatic fix.
```

Comment:
- [ Optional ] The `Getting Started` section of your readme file is very important, this means you always have to fill it correctly. Please see an example of how I would fill in this section in my `readme` file, the instructions you give in the `readme` file should always be clear and easy for someone to follow.

<!-- GETTING STARTED -->

## 💻 Getting Started <a name="getting-started"></a>

To get a local copy up and running, follow these steps.

```sh
   git clone git@github.com:muhammedb8826/Hello-world.git
```

### Prerequisites

In order to run this project you need:
 - NodeJS
 - Browser

### Setup

Clone this repository to your desired folder:

<!--
Example commands:
-->

```sh
  git clone git@github.com:muhammedb8826/Hello-world.git
  cd Hello-World
```

### Install

Install project dependencies in this project with:

```sh
  npm install
```

### Usage

To run the project, open the project with live server and visit [http://localhost:5500](http://localhost:5500)

### Open in browser

http://localhost:5500